source/CoordinateConversion/CoordinateConversionLibrary.Tests/CoordinateConversionLibrary.Tests.csproj

```diff
@@ -86,6 +111,9 @@
   </Choose>
   <Import Project="$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets" Condition="Exists('$(VSToolsPath)\TeamTest\Microsoft.TestTools.targets')" />
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
+  <PropertyGroup>
+    <PostBuildEvent>xcopy $(ProjectDir)json\*.* $(ProjectDir)$(OutDir)json /i /s /q /y</PostBuildEvent>
```

Comment:
This works - you could have also set this as a property of the files in Studio ("Copy to Output)